packages/graphql/src/scripts/2025-03-21_add_base_points_instance_options.ts

```diff
@@ -0,0 +1,42 @@
+import { PrismaClient } from '@klicker-uzh/prisma'
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
+dayjs.extend(utc)
+
+async function run() {
+  const prisma = new PrismaClient()
+
+  // count the number of element instances that are contained in the database
+  const count = await prisma.elementInstance.count()
+
+  // compute the number of batches of 100 instances
+  const batches = Math.ceil(count / 100)
+
+  // iterate over the batches of 100 instances and add the base points attribute with value true to the options
+  for (let i = 0; i < batches; i++) {
+    console.log(`Processing 100-instance batch ${i + 1}/${batches}`)
+
+    const instances = await prisma.elementInstance.findMany({
+      skip: i * 100,
+      take: 100,
+    })
+
+    for (const instance of instances) {
+      await prisma.elementInstance.update({
```

Comment:
I would just add a check that basePoints are only set to true if the property does not exist yet in the options (with any value). There should be no case matching that ever, but just to make sure, also if we ran this again in the future for any reason :)